### PR TITLE
Remove filter/search from App Directory widget

### PR DESCRIPTION
**Pull request by LeKeve**
## Description

This PR removes filter/search functionality from the App Directory widget. All available third party apps should now be listed on widget load.

JIRA ticket: https://vajira.max.gov/browse/API-2481

Ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/14279

Initial design guidelines: https://github.com/department-of-veterans-affairs/va.gov-team/issues/9238

## Testing done

- local testing on static pages

The widget can be tested locally by mounting to a static page that has an existing widget id. For example, to mount this widget to the `/find-forms` page
```
// src/applications/static-pages/widgetTypes.js
...
FIND_VA_FORMS: 'find-va-forms', // change to 'anything'
...
THIRD_PARTY_APP_DIRECTORY: 'third-party-app-directory', // change to 'find-va-forms'
...
```

## Screenshots

Screen Shot 2020-10-08 at 8 21 01 AM

## Acceptance criteria
- [x] filter/search removed widget
- [x] app results are shown on widget load

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ricetj** [comment]
Hey, @LeKeve do we want also want to remove the `showing 1-4 of results 4` as well. Since there is no filter at all? maybe that is something we add back after the filter is in place

cc: @bradenshipley

**Comment by LeKeve** [comment]
> Hey, @LeKeve do we want also want to remove the `showing 1-4 of results 4` as well. Since there is no filter at all? maybe that is something we add back after the filter is in place
> 
> cc: @bradenshipley

@ricetj 

That makes sense. I left it in there for now because in the context of a page with additional text/buttons it might be confusing where the list of apps ends. I'll hold off on changing until the new designs are in and mention this in the parent ticket if needed.

**Review by reedship — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]
You can add your team as a [codeowner](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/codeowners.md) of this app and then changes to it won't require a @department-of-veterans-affairs/frontend-review-group approval.

**Pull request merged**
